src/kvirc/ui/KviMainWindow.cpp

```diff
@@ -917,6 +918,17 @@ void KviMainWindow::contextMenuEvent(QContextMenuEvent *)
 	// do nothing! avoids builtin popup from qmainwindow
 }
 
+void KviMainWindow::keyPressEvent(QKeyEvent *event)
+{
+	switch(event->key())
+	{
+		case Qt::Key_Alt:
+			g_pMainWindow->showMenubar();
```

Comment:
should probably also include
```
else
case Qt::Key_Alt:
			g_pMainWindow->hideMenubar();
```

So you can also hide by pressing Alt, as is you need to click to hide